common/utilities.ts

```diff
@@ -224,7 +224,7 @@ export const TWENTY_FOUR_HOURS = 24 * 60 * 60 * 1000;
 
 export async function makeRequest({ endpoint }) {
   try {
-    const res = await fetch(`/api/${endpoint}`);
+    const res = await fetch(`https://internet-apis.onrender.com/api/${endpoint}`);
```

Comment:
can pass a host variable like this example shows: https://github.com/internet-development/apis/blob/main/app/page.tsx , otherwise looks good!